src/tools/ci/vfc_ci_report/compare_runs.py

```diff
@@ -0,0 +1,571 @@
+# Manage the view comparing a variable over different runs
```

Comment:
Please add a top level comment for future developpers to explain how to use datastructures and callbacks